codeguru_profiler_agent/sampling_utils.py

```diff
@@ -10,6 +10,7 @@
 TRUNCATED_FRAME = Frame(name="<Truncated>")
 
 TIME_SLEEP_FRAME = Frame(name="<Sleep>")
+LXML_SCHEMA_FRAME = Frame(name="lxml.etree:XMLSchema:__init__")
```

Comment:
Question: Should we follow the format of <...> for synthetic frame? For example, <lxml.etree:XMLSchema:__init__> here?

Reply:
Ok. We could always review this later :D.